keyboards/doio/kb04/info.json

```diff
@@ -0,0 +1,111 @@
+{
+    "manufacturer": "DOIO",
+    "keyboard_name": "KB04-01",
+    "maintainer": "filmstarr",
+    "bootloader": "stm32duino",
+    "build": {
+        "lto": true
+    },
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "B5", "pin_b": "B6"}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false,
+        "rgb_matrix": true
+    },
+    "matrix_pins": {
+        "cols": ["B14", "B13", "B12", "B0", "A7", "A9", "A8", "A6"],
```

Comment:
8 cols defined, mapped in rgb_matrix, but layout macro only exposes 5?

Reply:
Resolved in https://github.com/qmk/qmk_firmware/pull/22754/commits/69cd45e0d4c866bc98ba9d83d5d4caf798a129ab.